docker-compose.yaml

```diff
@@ -28,10 +28,10 @@ services:
         NEXT_PUBLIC_API_BASE_URL: "/api"
```

Comment:
why are we uncommenting volumes? this was commented earlier due to some error as far as I remember